README.md

```diff
@@ -69,6 +69,88 @@ We're now on the Maven Central Repository. Add the following to your POM file:
 
 The driver can be downloaded from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774)
 
+##Dependencies
+This project has following dependencies: 
+
+Compile Time:
+
+ - `azure-keyvault`
```

Comment:
Can we add what this dependency is used for?  E.g. used for Azure Key Vault Provider for Always Encrypted